solutions/account-infrastructure-base/variables.tf

```diff
@@ -39,3 +39,9 @@ variable "cos_bucket_management_endpoint_type" {
     error_message = "The specified management_endpoint_type_for_bucket is not a valid selection!"
   }
 }
+
+variable "allowed_ip_addresses" {
+  description = "List of the IP addresses and subnets from which IAM tokens can be created for the account."
+  type        = list(string)
+  default     = ["0.0.0.0/0"]
```

Comment:
Do not default to this! Intact this should be a required variable, so remove all defaults. We can pass this value in the TEST, no here

Reply:
If its required, you will also need to add it to the cra-config.yaml and catalogValidationValues.json.template

Reply:
I've updated the variable to no longer have a default and passed it in to the test/cra/catalog validation, I wasn't sure on formatting for a list when passing as a variable in the yaml so I left it as a string but if it needs to be changed to a yaml list I can update it